tests/e2e/volume_health_test.go

```diff
@@ -847,16 +832,6 @@ var _ = ginkgo.Describe("Volume health check", func() {
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		}()
 
-		ginkgo.By(fmt.Sprintf("Invoking QueryCNSVolumeWithResult with VolumeID: %s", volHandle))
```

Comment:
why is this removed?

Reply:
This is not required now. 
earlier test was allowing to edit the Volume health annotation  to any value. after some time , when volume health check gets triggered , it automatically resetting the annotation value back to the original value. So we had multiple steps to validate the annotation status , CNSVolume, etc... Now it will not allow to edit the annotation itself. It will throw error

Reply:
volume health in CNS is different from k8s, we should verify it is green